src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ThemeManager.cs

```diff
@@ -469,9 +470,11 @@ private static bool IsSystemThemeLight()
 
 
     #region Private Fields
-    private static readonly string fluentThemeResoruceDictionaryUri = "pack://application:,,,/PresentationFramework.Fluent;component/Themes/";
-    private static readonly string _regPersonalizeKeyPath = "HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize";
-    private static FluentThemeState _currentFluentThemeState = new FluentThemeState("None", false, SystemColors.AccentColor);
+    private const string _fluentThemeResoruceDictionaryUri = "pack://application:,,,/PresentationFramework.Fluent;component/Themes/";
+    private const string _regPersonalizeKeyPath = "HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize";
+    private const string _fluentLightDictionary = "Fluent.Light.xaml";
+    private const string _fluentDarkDictionary = "Fluent.Dark.xaml";
```

Comment:
Constants should be PascalCase in .NET.

Reply:
Reference: https://github.com/dotnet/runtime/blob/1a811ea83965a3ce41ae573a6265b557d7662113/docs/coding-guidelines/coding-style.md

> 12. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.